semgrep/semgrep/cli.py

```diff
@@ -282,6 +290,35 @@ def cli() -> None:
         error_on_findings=args.error,
     )
 
+    if not args.disable_version_check:
+        quick_timeout = 2  # Don't block user's too long
+        endpoint = os.environ.get(
+            "VERSION_CHECK_URL", "https://semgrep.live/api/check-version"
+        )
+        try:
+            resp = requests.get(
```

Comment:
I tested this manually offline as well. The nice thing is it instantly returns in that case and doesn't wait for the timeout.